### DISCUSSION TITLE
'Could not find exact match level' Error for TF adjustment

### DISCUSSION THREAD

**Discussion by richard-a-lott** [moj-analytical-services/splink#1606] [Q&A]
I'm getting errors when trying to generate TF adjustments for a name comparison. The error is: "ValueError: Could not find exact match level for cleansed_full_name"

I have 3 fields relating to this; Firstnames, surname and a combined full name field. I'm defining a custom comparison with:

- Null level
- Full name exact match and first names not missing (in order to separate out surname only matches)
- Surname exact match and first names missing (in order to get ONLY surname matches) 
- First names exact match and surnames missing (ONLY first names match)
- Some fuzzy levels & else level

Here's a snip of the full name exact match for ref:
![image](https://github.com/moj-analytical-services/splink/assets/136445689/bdaee72c-1192-44f6-8400-37dc00caa8e0)

Note that I have TF adjustment fields referenced in the first & surname match levels too.

How is splink determining what is/is not an exact match level? I tried finding an equivalent of
`"null_level": True `
but I can't see anything like that for exact match.

I also tried different labels but that didn't work either.

What am I doing wrong? Am I trying to use TF adjustments in an inappropriate way?

**Comment by RobinL** [comment]
Hmm, I think this may be a bug.  Are you able to share the settings for the whole Comparison so we can try to reproduce and fix

**Reply by richard-a-lott** [reply]
Sure. I can only share it as an image I'm afraid, but here you go:
![image](https://github.com/moj-analytical-services/splink/assets/136445689/7e408030-7872-44eb-a0ff-8638c6faacc1)

cleansed_full_name = cleansed forename + " " + cleansed_surname
